scaleway/resource_lb_test.go

```diff
@@ -69,37 +69,6 @@ func TestAccScalewayLbLb_WithIP(t *testing.T) {
 		ProviderFactories: tt.ProviderFactories,
 		CheckDestroy:      testAccCheckScalewayLbDestroy(tt),
 		Steps: []resource.TestStep{
-			{
```

Comment:
Why is this step removed?

Reply:
This test is now considered as an error. This configuration will always have an private network attachement state as an error.